peer/brontide.go

```diff
@@ -1455,6 +1455,10 @@ out:
 				break out
 			}
 
+		case *lnwire.Warning:
+			targetChan = msg.ChanID
+			isLinkUpdate = p.handleError(&msg.Error)
```

Comment:
On second thought, maybe we actually _shouldn't_ send this error to the link/fundingMgr? This'll cause the link to disconnect, though I guess our behavior of just tearing down the link (and _not_ force closing) is now the intended behavior?